Assets/Scripts/Enemies/EnemyController.cs

```diff
@@ -47,4 +47,9 @@ bool IsOutOfView(Vector3 position)
         return position.x >= WorldCoordinates.LargestDimension || position.x <= -WorldCoordinates.LargestDimension
             || position.y >= WorldCoordinates.LargestDimension || position.y <= -WorldCoordinates.LargestDimension;
     }
+
+    public Spawner getSpawner()
```

Comment:
The `Spawner` is set in `EnemyMovement`. I think it makes more sense to just make it a protected variable there rather than create a method here to return it.